### PR TITLE
Hw_10_Bloggy.app

### PR DESCRIPTION
**Pull request by Lansaro**
Little buggy......

### CODE REVIEW

File: LeoTishin-Lansaro/hw_10/bloggy_app/test/models/blog_test.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# == Schema Information
```
Comment (tgaff) [inline LeoTishin-Lansaro/hw_10/bloggy_app/test/models/blog_test.rb]: looks like you got annotate is working, good!

**Comment by tgaff** [comment]
Bootstrap looks good and it seems to work well.
  But I don't see comments... did you finish the getting started guide?

**Comment by Lansaro** [comment]
Hey, I've finished the Bloggy app

**Comment by tgaff** [comment]
Looks good Leo.  I think you're still missing the delete a comment feature, but that's it.  Did you have much luck figuring out the html/bootstrap stuff?

**Pull request merged**